.c8rc.json

```diff
@@ -1,3 +1,5 @@
 {
-  "all": "true"
```

Comment:
"all" is required.
Otherwise those files which are not introduced will be ignored. For example, if you delete `test/highlight.spec.ts`, you will find that the coverage reaches 100